src/test/daml/Daml/Finance/Instrument/Option/Test/EuropeanPhysical.daml

```diff
@@ -59,56 +59,56 @@ run = script do
   let
     expectedConsumed = [qty 40.0 cashInstrumentCid]
     expectedProduced = [qty 1.0 aaplInstrumentCid]
-  optionInstrumentAfterFirstPayment <- electAndVerifyPaymentEffects [publicParty] expiryDate amount
-    itmLongCallInstrument longOption issuer custodian "EXERCISE" expectedConsumed expectedProduced
+  optionInstrumentAfterFirstPayment <- electAndVerifyPaymentEffects expiryDate amount
+    itmLongCallInstrument longOption issuer custodian custodian "EXERCISE" expectedConsumed expectedProduced
```

Comment:
On a long call, I would expect the `owner` to be the elector rather than the `custodian`?

Reply:
This is a unit-style test, where we typically do not have any holdings, just instruments and effects. There is not even an investor / owner party. I guess that was fine for instruments with `Evolve` based lifecycling, but for `Election` based lifecycling maybe we should add an investor / owner as well?

Reply:
An `Election` is tied to a specific party being the `elector`. In most cases, the `elector` is the owner of the instrument. 

I think it makes sense to use an `owner / investor` party, otherwise this code happens to be misleading (it would not work in practice). 

In the case of elections, we might consider extending the notion of "unit-style" test to include claiming the effect, as that is integral to the election process.

Reply:
> In the case of elections, we might consider extending the notion of "unit-style" test to include claiming the effect, as that is integral to the election process.

I think that makes sense. I have opened a separate issue: https://github.com/digital-asset/daml-finance/issues/706

Reply:
> I think it makes sense to use an `owner / investor` party, otherwise this code happens to be misleading (it would not work in practice).

I agree. I have added this now.